BinaryTreeRange.java

```diff
@@ -0,0 +1,70 @@
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class BinaryTreeRange {
+    public static void main(String[] args) {
+        test(new int[]{1,2,3,4,5}, 1, 5);
+        test(new int[]{}, 1, 5);
+        test(new int[]{1,2,3,4,5}, 5, 3);
+    }
+
+    public static int findNumbersInRange(Node root, int start, int end) {
+        int result = 0;
+        Queue<Node> notVisited = new LinkedList<>();
+        if(root == null) return 0;
```

Comment:
Nit: this should be done before creating the Queue to avoid creating the queue in this case.